framework/source/class/qx/io/remote/Rpc.js

```diff
@@ -568,7 +568,18 @@ qx.Class.define("qx.io.remote.Rpc",
             break;
 
           case 1: // async with handler function
-            handler(result, ex, id);
+            try
+            {
+              handler(result, ex, id);
+            }
+            catch(e)
+            {
+              console.error(
```

Comment:
shouldn't this be this.error instead of console?  Im not sure that console is always available (although that may be an IE8 thing Im remembering), but this.error allows for filtering and other outputs